license.txt

```diff
@@ -0,0 +1,7 @@
+Copyright 2019 Novom Interactive
```

Comment:
Name of the file should be in all caps (LICENSE.txt). Also, "(c)" should be added after the word "Copyright".